### PR TITLE
fixed the interviewee's name color w.r.t the bg color theme

### LINKED ISSUE #5314: [Bug] Name of the Maintainers isn't visible in dark mode.
### Description
In _Meet The Maintainer_ Blog series, the maintainer's name is not visible in dark mode.

### Expected Behavior
Rendering needs to be fixed.

### Screenshots
Example used for screenshots : [here](https://layer5.io/blog/community/meet-the-maintainer-michael-gfeller)
image
image

### Environment:
- Host OS: Windows
- Browser: Chrome

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Abraham599**
**Description**
now the interviewee's name (Michael) will change to the right font color w.r.t whenever the background theme changes. 
This PR fixes #5314

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

when the bg theme is dark
image

when the bg theme is white in color
image

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 75926fda78511d45f3a6fc02c5a58f421e3e5002 at: https://65af888b12f1c3c6ec9ac491--layer5.netlify.app

File: src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js [hunk 1]
```diff
@@ -288,8 +288,8 @@ const IntegrationsGrid = ({ category, count }) => {
                           : integrationIcon
                       }
                       alt={item.frontmatter.title}
-                      height={70}
-                      width={70}
+                      height={55}
```
Comment (sudhanshutech) [inline src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js:291]: why this change?

File: src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js [hunk 2]
```diff
@@ -316,8 +316,8 @@ const IntegrationsGrid = ({ category, count }) => {
                             : integrationIcon
                         }
                         alt={item.frontmatter.title}
-                        height={70}
-                        width={70}
+                        height={55}
```
Comment (sudhanshutech) [inline src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js:319]: ?

File: src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js (hunk 1 above)
Comment (Abraham599) [reply src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js:291]: Actually I had make those changes for fixing the height and width AMD logo in the integration grid section page. So, this was for a prior issue for which I was assigned for. 

This was the issue 

[Bug] Name of the Maintainers isn't visible in dark mode. #5314


I am new to open source contribution so correct me if I had not followed the guidelines well.

**Comment by saurabh100ni** [comment]
Hi @Abraham599  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.
